### PR TITLE
Add pull request for personal website

### PR DESCRIPTION
**Pull request by vaupunkt**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        </p>
+      </section>
+      <section id="contact">
+        <h2>Contact</h2>
+      </section>
+      <section id="biography">
+        <h2>Biography</h2>
+      </section>
+    </main>
+    <footer></footer>
```
Comment (ahohnsen) [inline index.html]: If you don't need it, delete it 🙂 🧹

**Review by ahohnsen — APPROVED** [review]
Well done 💪🏼 Left a little comment.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Veliko Kardziev</title>
+  </head>
+  <body>
+    <header>
+      <h1>Veliko Kardziev</h1>
+      <nav>
+        <a href="#aboutme">About me</a>
+        <a href="contact">Contact</a>
```
Comment (josch87) [inline index.html:14]: you probably wanted to reference an id

File: index.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        <a href="biography">Biography</a>
```
Comment (josch87) [inline index.html:15]: you probably wanted to reference an id

**Review by josch87 — COMMENTED** [review]
2 bugs in the links, looks good besides that :)

File: index.html (hunk 2 above)
Comment (makitango) [reply index.html:14]: Agreed. #contact would make sense here

**Pull request merged**